### DISCUSSION TITLE
Frame won't always run

### DISCUSSION THREAD

**Discussion by Tempy111** [arkypita/LaserGRBL#2101] [Q&A] [answered]
I have a Atomstack P9 M40 and while it works fine with laser GRBL, there is the odd problem. Can't quite get the 'focus' button to work but not too bothered (fixed focus laser) but the problem is the frame. Sometimes, I can use the frame and it'll do the axles movement to outline the frame of the cut/engrave correctly. Other times it comes up erroring that it's spool rate isn't right or isn't set (Sorry, I need to take a screenshot but I use a different machine for controlling it). This is a bit of a problem when it comes to trying to make sure the material is aligned correctly.

Anyone know any reason why this might happen sometimes? I haven't really figured out why it sometimes works but more often then not doesn't

**Comment by Tempy111** [comment]
Update: the script it runs brings up:
M3 S[$30*3/100] F1000 - Bad Number Format
then the movement commands undefined feed rate 

![badNumber](https://github.com/arkypita/LaserGRBL/assets/21968380/c1564d4f-90b8-4081-b8ef-d0dc9728c246)

**Reply by arkypita** [reply, marked as answer]
Please open menu grbl - grbl configuration and let LaserGRBL read machine configuration. Then try again

**Reply by Tempy111** [reply]
ah, thanks. works perfectly ^_^
